cookiecutter.json

```diff
@@ -5,5 +5,9 @@
   "author": "Claudio Jolowicz",
   "email": "mail@claudiojolowicz.com",
   "github_user": "cjolowicz",
-  "version": "0.0.0"
+  "version": "0.0.0",
+  "license": ["MIT", "Apache-2.0", "GPL-3.0"],
+  "year": "{% now 'utc', '%Y' %}",
+  "full_license_name": "{{ {'MIT': 'MIT License', 'Apache-2.0': 'Apache Licence 2.0', 'GPL-3.0': 'Gnu General Public License v3.0'}.get(cookiecutter.license, '') }}",
+  "_extensions": ["jinja2_time.TimeExtension"]
```

Comment:
This is the line Prettier has issues with, but it seems to match the desired output.

Reply:
This is incorrect:

> [Prettier is complaining about the generated .cookiecutter.json in the project itself, as opposed to the template's cookiecutter.json.](https://github.com/cjolowicz/cookiecutter-hypermodern-python/pull/701#issuecomment-745587106)